//package com.sincetimes.redwar.game.comnunicate.protocol;
message RseQueryReceiveWishItem
{
	optional int32 ret = 1;
}